OnepayMVCTest/Controllers/TransactionController.cs

```diff
@@ -1,19 +1,28 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using Newtonsoft.Json.Serialization;
 using Newtonsoft.Json;
 using System.Web.Mvc;
 using OnepayMVCTest.Models;
 using Transbank.Onepay.Model;
 using System.Diagnostics;
+using Transbank.Onepay.Enums;
 
 namespace OnepayMVCTest.Controllers
 {
     public class TransactionController : Controller
     {
         [HttpPost]
-        public ActionResult Create()
+        public JsonResult Create(string channel)
         {
+            var channelType = ChannelType.Web;
+
+            if (channel.Equals("mobile", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Can we encapsulate this logic in the `ChannelType` class in the SDK? So that the integrator only has to do  `var channelType = ChannelType.FromString(channelType)` and we avoid this code block from the example (and from every checkout integration).

Reply:
@goncafa This can go in a new PR BTW so that we don't block the example for onepay

Reply:
@goncafa regarding the feature to implement, it would be `ChannelType.Parse(channel)` rather than `ChannelType.FromString()`. That way we match the documentation at https://transbankdevelopers.cl/documentacion/onepay?csharp#2-back-end-crear-una-transacci-n :).